packages/xo-web/src/xo-app/logs/log-alert-header.js

```diff
@@ -46,6 +71,15 @@ export default [
             title='Backup job failed'
           />
         )}
+        {!isEmpty(state.failedVmsIds) &&
```

Comment:
Why use `isEmpty` instead of testing the `length`?

Reply:
I will test the length instead.